### ISSUE TITLE
Contact fastQC people

### ISSUE THREAD

**Issue by Kiwwa** [starcalibre/fastQC-viz#9]
As Bernie suggested, we should contact the fastQC peeps and see if they're interested in the work we're doing, whether they have suggestions for features/pipeline'ing/whatever and anything else.

Their contact page is;
http://www.bioinformatics.babraham.ac.uk/people.html

**Comment by starcalibre** [comment]
Agreed! Maybe let's finish the view multiple files functionality so we have
a live demo before we do that?

**Comment by Kiwwa** [comment]
Sounds good to me :+1:
